core-js/src/main/javascript/cdf/components/filter/controllers/Manager.js

```diff
@@ -210,13 +210,17 @@ define([
      * Pagination
      */
     getNextPage: function(model, event) {
-      var ref;
-      this.previousPosition = (ref = _.last(this.sortChildren(this.children()), 2)[0]) != null ? ref.get('view').$el : void 0;
+      var listOfChildren = this._listChildren(this.children());
+      var sortedChildren = this.sortChildren(listOfChildren);
+      var penultimateChild = _.last(sortedChildren, 2)[0];
+      this.previousPosition = penultimateChild != null ? penultimateChild.target : undefined;
       return this.getPage('next', model, event);
     },
     getPreviousPage: function(model, event) {
-      var ref;
-      this.previousPosition = (ref = _.first(this.sortChildren(this.children()), 2)[1]) != null ? ref.get('view').$el : void 0;
+      var listOfChildren = this._listChildren(this.children());
+      var sortedChildren = this.sortChildren(listOfChildren);
+      var penultimateChild = _.last(sortedChildren, 2)[1];
```

Comment:
note that in this case you want the second child:
```javascript
var secondChild = _.first(sortedChildren, 2)[1];
```
or even 
var secondChild = sortedChildren.length > 1 ?  sortedChildren[1] : null;